index.html

```diff
@@ -0,0 +1,175 @@
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Controls</h3>
+          <div class="card-body">
+            <h6>Bot:</h6>
+            <button type="button" onclick="startBot()" class="btn btn-success">
+              start
+            </button>
+            <button type="button" onclick="stopBot()" class="btn btn-danger">
+              stop</button
+            ><br />
+            <h6 class="mt-4">Websocket:</h6>
+            <button
+              type="button"
+              onclick="startWebSocket()"
+              class="btn btn-success"
+            >
+              open
+            </button>
+            <button
+              type="button"
+              onclick="closeWebSocket()"
+              class="btn btn-outline-danger"
+            >
+              close
+            </button>
+          </div>
+        </div>
+      </div>
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Status</h3>
+          <div class="card-body">
+            <span class="dark-text">Coordinates:</span>
+
+            <span id="coords" class="badge badge-primary">0 0 0</span><br />
+
+            <span class="dark-text">Health: </span>
+            <span id="health" class="badge badge-danger">20</span><br />
+            <span class="dark-text">Hunger: </span>
+            <span id="hunger" class="badge badge-warning">20</span><br />
+          </div>
+        </div>
+      </div>
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Settings</h3>
+          <div class="card-body">
+            <form role="form">
+              <div class="form-group">
+                <label class="dark-text">Server address</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="server"
+                  placeholder="mc.hackclub.com"
+                />
+              </div>
+              <div class="form-group">
+                <label for="exampleInputPassword1" class="dark-text"
+                  >Port</label
+                >
+                <input
+                  type="number"
+                  class="form-control"
+                  id="port"
+                  placeholder="25565"
+                />
+              </div>
+              <button
+                type="button"
+                class="btn btn-outline-primary"
+                onclick="setServer()"
+              >
+                Set
+              </button>
+            </form>
+          </div>
+        </div>
+      </div>
+
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Movement</h3>
+          <div class="card-body">
+            <div class="sprintStatus">
+              <span class="badge badge-primary" id="sprintStatus">
+                Walking
+              </span>
+            </div>
```

Comment:
We aren't detecting `sprintStatus` at the moment, so I think we're fine to remove this for now
```suggestion
```